content/sensu-go/5.21/release-notes.md

```diff
@@ -63,6 +64,35 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.21.0 release notes
+
+**June 10, 2020** &mdash; The latest release of Sensu Go, version 5.21.0, is now available for download.
+
+**RELEASE SYNOPSIS NEEDED HERE**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.21.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][158]) Added [entity count and limit][156] for each entity class in the tabular title in the response for `sensuctl license info` (in addition to the total entity count and limit).
+- ([Commercial feature][158]) Added [Linux amd64 OpenSSL-linked binaries][160] for the Sensu agent and backend, with accompanying `--require-fips` and `--require-openssl` flags for the [agent][161] and [backend][162].
+- Added `sensuctl user hash-password` command to generate password hashes.
+- Added the ability to reset passwords via the backend API and `sensuctl user reset-password`.
+- Added binary distribution for the Sensu agent for the [AIX PPC64][159] platform.
```

Comment:
We won't be releasing AIX builds for 5.21 as we ran into some technical difficulties.